hedera-state-validator/README.md

```diff
@@ -44,10 +44,29 @@ of a corrupted state.
 2. Run the following command to execute the introspection:
 
    ```shell
-   java -jar ./validator-<version>.jar introspect {serviceName} {stateName} [{keyInfo}]
+   java -jar ./validator-<version>.jar {path-to-state-round} introspect {serviceName} {stateName} [{keyInfo}]
```

Comment:
Maybe change "stateName" term to "stateKey" for consistency, as "stateKey" is the term which is used in prod code?

Reply:
Provide a few examples in the doc, maybe?